src/System.Management.Automation/engine/SessionState.cs

```diff
@@ -305,6 +306,89 @@ internal void InitializeFixedVariables()
                     RunspaceInit.PSHostDescription);
             this.GlobalScope.SetVariable(v.Name, v, asValue: false, force: true, this, CommandOrigin.Internal, fastPath: true);
 
+            // $PSClipboardActions
+            Hashtable clipboardActions = null;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
I really don't like that I stuffed all the platform logic in this file specifically, but I couldn't find a better place; is this good?